### ISSUE TITLE
New Relic sample code uses old API

### ISSUE THREAD

**Issue by hoff331** [acquia/cloud-hooks#51]
The New Relic sample hook, newrelic.sh, uses the old API and per [NewRelic](https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/recording-deployments#post-deployment) should read:

```
curl -X POST 'https://api.newrelic.com/v2/applications/${APP_ID}/deployments.json' \
     -H 'X-Api-Key:${API_KEY}' -i \
     -H 'Content-Type: application/json' \
     -d \
'{
  "deployment": {
    "revision": "REVISION",
    "changelog": "Added: /v2/deployments.rb, Removed: None",
    "description": "Added a deployments resource to the v2 API",
    "user": "datanerd@example.com"
  }
}' 
```

Using what already exists in the newrelic.sh file, we can replace the strings with the variables we have available to us:

```
#https://docs.newrelic.com/docs/apm/new-relic-apm/maintenance/recording-deployments#post-deployment
curl -X POST 'https://api.newrelic.com/v2/applications/$APPID/deployments.json' \
     -H 'X-Api-Key:$APIKEY' -i \
     -H 'Content-Type: application/json' \
     -d \
'{
  "deployment": {
    "revision": "$deployedtag",
    "changelog": "$deployedtag deployed to $site:$targetenv",
    "description": "$deployedtag deployed to $site:$targetenv",
    "user": "$username"
  }
}'
```

**Comment by weitzman** [comment]
This has been fixed.

**Comment by blkperl** [comment]
Fixed in #52
